ginga/rv/plugins/MultiDim.py

```diff
@@ -288,6 +304,7 @@ def play_axis_change():
                         self.play_max = dims[self.play_axis]
 
                     self.play_idx = self.play_indices[n - 2]
+                    self.play_last[imname] = self.play_idx
```

Comment:
I think it should be possible to tell the slice from the `naxispath` attribute in the `AstroImage`.  I think this would be preferable to remembering which was the last item played through `MultiDim` (because there are other ways that the slice can be set externally to `MultiDim`).  This would also have the advantage that the information would persist even if `MultiDim` is closed and reopened.  And it would simplify the code here.